docs/content/architecture/postgres-operator-containers-overview.md

```diff
@@ -36,9 +36,9 @@ The PostgreSQL Operator orchestrates a series of PostgreSQL and PostgreSQL relat
 
 ### Metrics and Monitoring
 
-* **Metrics Collection** (crunchy-collect). The crunchy-collect container provides real time metrics about the PostgreSQL database via an API. These metrics are scraped and stored by a Prometheus time-series database and are then graphed and visualized through the open source data visualizer Grafana.  
+* **Metrics Collection** (crunchy-postgres-exporter). The crunchy-postgres-exporter container provides real time metrics about the PostgreSQL database via an API. These metrics are scraped and stored by a Prometheus time-series database and are then graphed and visualized through the open source data visualizer Grafana.  
 
-* **Grafana** (crunchy-grafana).  Visual dashboards are created from the collected and stored data that crunchy-collect and crunchy-prometheus provide for the crunchy-grafana container, which hosts an open source web-based graphing dashboard called Grafana.
+* **Grafana** (crunchy-grafana).  Visual dashboards are created from the collected and stored data that crunchy-postgres-exporter and crunchy-prometheus provide for the crunchy-grafana container, which hosts an open source web-based graphing dashboard called Grafana.
```

Comment:
This whole sentence seems awkward. The one in #1757 looks good to me and doesn't include container names.

Perhaps revert this change so that it doesn't conflict during merge?

Reply:
No problem. I haven't actually rebased to include 1757 yet, so I can get that when I do.